src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam
+-/
+import analysis.special_functions.trigonometric
+
+open real
```

Comment:
To maintainers/reviewers, should everything in this file be in the root namespace (which it currently is) or should it be in the `real` namespace?

Reply:
If so, please mark `arsinh` as `@[pp_nodot]`

Reply:
It's all about real.arsinh right? I havent dealt with its complex counterpart, if it even exists.

Reply:
Complex inverse hyperbolic sine is conventionally defined to have branch cuts on the imaginary axis outside the interval from -i to i (and to be continuous elsewhere and to agree with the real function for real arguments). See https://dlmf.nist.gov/4.37 for the conventional definitions of the principal values of inverse hyperbolic functions for complex arguments (and some other basic properties of those functions).

Reply:
Done! I've added namespaces and the `@[pp_nodot]` tag